trigger_webapp/trigger_app/models.py

```diff
@@ -24,20 +25,30 @@ class Telescope(models.Model):
+        return f"{self.name}"
+
+
 class ProposalSettings(models.Model):
     id = models.AutoField(primary_key=True)
     #telescope = models.CharField(max_length=64, blank=True, null=True, verbose_name="Telescope name", help_text="E.g. MWA_VCS, MWA_correlate or ATCA. If the telescope you want is not here add it on the admin page.")
     telescope = models.ForeignKey(Telescope, to_field="name", verbose_name="Telescope name", help_text="Telescope this proposal will observer with. If the telescope you want is not here add it on the admin page.", on_delete=models.CASCADE)
-    project_id = models.CharField(max_length=64, blank=True, null=True, help_text="This will be used to schedule observations.")
-    proposal_description = models.CharField(max_length=256, blank=True, null=True, help_text="A brief description of the proposal. Only needs to be enough to distinguish it from the other proposals.")
-    trig_min_duration = models.FloatField(blank=True, null=True, verbose_name="Min")
-    trig_max_duration = models.FloatField(blank=True, null=True, verbose_name="Max")
-    pending_min_duration = models.FloatField(blank=True, null=True, verbose_name="Min")
-    pending_max_duration = models.FloatField(blank=True, null=True, verbose_name="Max")
-    fermi_prob = models.FloatField(blank=True, null=True, help_text="The minimum probability to observe for Fermi sources (it appears to be a percentage, e.g. 50).")
-    swift_rate_signf = models.FloatField(blank=True, null=True, help_text="The minimum \"RATE_SIGNIF\" (appears to be a signal-to-noise ratio) to observe for SWIFT sources (in sigma).")
-    repointing_limit = models.FloatField(blank=True, null=True, help_text="An updated position must be at least this far away from a current observation before repointing (in degrees).")
-    horizon_limit = models.FloatField(blank=True, null=True, help_text="The minimum elevation of the source to observe (in degrees).")
+    project_id = models.CharField(max_length=64, help_text="This will be used to schedule observations.")
+    proposal_description = models.CharField(max_length=256, help_text="A brief description of the proposal. Only needs to be enough to distinguish it from the other proposals.")
+    event_telescope = models.ForeignKey(EventTelescope, to_field="name", help_text="The telescope that this proposal will accept at least one VOEvent from before observing. Leave blank if you want to accept all telescopes.", blank=True, null=True, on_delete=models.SET_NULL)
+    trig_min_duration = models.FloatField(verbose_name="Min", default=0.256)
+    trig_max_duration = models.FloatField(verbose_name="Max", default=1.024)
+    pending_min_duration_1 = models.FloatField(verbose_name="Min", default=1.025)
+    pending_max_duration_1 = models.FloatField(verbose_name="Max", default=2.056)
+    pending_min_duration_2 = models.FloatField(verbose_name="Min", default=0.128)
+    pending_max_duration_2 = models.FloatField(verbose_name="Max", default=0.255)
```

Comment:
This is because I use fieldsets to put a title and description on top of all of these pairs.
```
        ("Source Settings: Trigger Duration Range (s)", {
            'fields':(
                ('trig_min_duration', 'trig_max_duration'),
            ),
            'description': "The inclusive duration range of an event that will automatically trigger an observation.",
        }),
```
Which looks like this:
![Screenshot from 2022-02-24 13-43-53](https://user-images.githubusercontent.com/18390846/155465290-0058d6bb-c173-4894-83f2-b178de338f96.png)